chromium_src/chrome/common/url_constants.h

```diff
@@ -277,7 +277,7 @@ inline constexpr char kTabGroupsLearnMoreURL[] = "https://support.brave.com/";
 
 // The URL for the Learn More page about policies and enterprise enrollment.
 inline constexpr char16_t kManagedUiLearnMoreUrl[] =
-    u"https://support.brave.com/";
+    u"https://support.brave.com/hc/en-us/articles/360039248271-Group-Policy";
```

Comment:
Will this be overwritten on the next chromium update @mkarolin?

Reply:
We maintain this source manually, if your question is about whether we have some tool that could overwrite this file.